lib/web_ui/dev/create_simulator.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:async';
+
+import 'package:args/command_runner.dart';
+import 'package:simulators/simulator_manager.dart';
+
+import 'safari_installation.dart';
+import 'utils.dart';
+
+class CreateCommand extends Command<bool> with ArgUtils {
+  CreateCommand() {
+    IosSafariArgParser.instance.populateOptions(argParser);
+  }
+
+  @override
+  String get name => 'create';
+
+  @override
+  String get description => 'Creates mobile simulators.';
+
+  @override
+  FutureOr<bool> run() async {
+    IosSafariArgParser.instance.parseOptions(argResults);
+    final IosSimulatorManager iosSimulatorManager = IosSimulatorManager();
+    try {
+    final IosSimulator simulator = await iosSimulatorManager.createSimulator(
+        IosSafariArgParser.instance.iosMajorVersion,
+        IosSafariArgParser.instance.iosMinorVersion,
+        IosSafariArgParser.instance.iosDevice);
+    print('INFO: Simulator created ${simulator.toString()}');
```

Comment:
there's an indent issue in this try/catch block

Reply:
done. I use document format.